From c5f39159e594637f9a81424d60375c23ff86908b Mon Sep 17 00:00:00 2001
From: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Date: Tue, 11 Mar 2014 13:39:33 +0000
Subject: [PATCH 149/351] ASoC: florida: Apply rev E patch to future devices

These will still be required whilst the DRE is disabled.

Change-Id: Ia69321461a97c44461647e0cb33806883ac78034
Signed-off-by: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Signed-off-by: Nikesh Oswal <Nikesh.Oswal@wolfsonmicro.com>
---
 sound/soc/codecs/florida.c |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/sound/soc/codecs/florida.c b/sound/soc/codecs/florida.c
index a4971ff..cb2618f 100644
--- a/sound/soc/codecs/florida.c
+++ b/sound/soc/codecs/florida.c
@@ -194,12 +194,10 @@ static int florida_sysclk_ev(struct snd_soc_dapm_widget *w,
 		patch = florida_sysclk_revd_patch;
 		patch_size = ARRAY_SIZE(florida_sysclk_revd_patch);
 		break;
-	case 4:
+	default:
 		patch = florida_sysclk_reve_patch;
 		patch_size = ARRAY_SIZE(florida_sysclk_reve_patch);
 		break;
-	default:
-		return 0;
 	}
 
 	switch (event) {
-- 
1.7.9.5

